network_modeling/configs/templates/1.3/aruba/common/sw-cdu.primary.j2

```diff
@@ -47,37 +79,66 @@ interface mgmt
 
 interface loopback 0
     ip address {{ variables.LOOPBACK_IP }}/32
+    ipv6 address  {{ variables.LOOPBACK_IPv6 }}/128
     ip ospf 1 area 0.0.0.0
+    ipv6 ospfv3 1 area 0.0.0.0
+interface loopback 1
+    ipv6 address  {{ variables.LOOPBACK_IPv6 }}/128
+    ipv6 ospfv3 2 area 0.0.0.0
 interface vlan 1
     ip mtu 9198
     ip address {{ variables.MTL_IP }}/{{variables.MTL_PREFIX_LEN}}
+    ipv6 address {{ variables.MTL_IPv6 }}/{{variables.MTL_PREFIX_LEN_IPv6}}
     ip ospf 1 area 0.0.0.0
     ip ospf passive
+    ipv6 ospfv3 1 area 0.0.0.0
+    ipv6 ospfv3 passive
+    ip igmp enable
+    ipv6 mld enable
 interface vlan {{ variables.NMN_VLAN }}
     description NMN
     ip mtu 9198
     ip address {{ variables.NMN_IP }}/{{variables.NMN_PREFIX_LEN}}
+    ipv6 address {{ variables.NMN_IPv6 }}/{{variables.NMN_PREFIX_LEN_IPv6}}
     ip ospf 1 area 0.0.0.0
+    ipv6 ospfv3 1 area 0.0.0.0
+    ip igmp enable
+    ipv6 mld enable
 interface vlan {{ variables.HMN_VLAN }}
     description HMN
     ip mtu 9198
     ip address {{ variables.HMN_IP }}/{{variables.HMN_PREFIX_LEN}}
+    ipv6 address {{ variables.HMN_IPv6 }}/{{variables.HMN_PREFIX_LEN_IPv6}}
     ip ospf 1 area 0.0.0.0
     ip ospf passive
+    ipv6 ospfv3 1 area 0.0.0.0
+    ipv6 ospfv3 passive
+    ip igmp enable
+    ipv6 mld enable
 interface vlan {{ variables.CMN_VLAN }}
     vrf attach Customer
     description CMN
     ip mtu 9198
     ip address {{ variables.CMN_IP }}/{{variables.CMN_PREFIX_LEN}}
+    ipv6 address {{ variables.CMN_IPv6 }}/{{variables.CMN_PREFIX_LEN_IPv6}}
     ip ospf 2 area 0.0.0.0
-
+    ipv6 ospfv3 2 area 0.0.0.0
+    ip igmp enable
+    ipv6 mld enable
+snmp-server vrf default
 ip dns server-address {{ variables.NMNLB_DNS }}
 router ospf 2 vrf Customer
     router-id {{ variables.LOOPBACK_IP }}
     area 0.0.0.0
+router ospfv3 2 vrf Customer
+    router-id {{ variables.LOOPBACK_IP }}
```

Comment:
Use the same v4 Loopback ID?

Reply:
no, we would need new unique IP for the second loopback interface. Its for mcast routing. 

I think this is something I need to add on my list of Variables (I am documenting them as part of PR: 

Phase 6: Document initial IPv6 design (docs-csm)

https://jira-pro.its.hpecorp.net:8443/browse/CASMNET-1647

Reply:
to be resolved as part of https://jira-pro.its.hpecorp.net:8443/browse/CASMNET-1665